src/assets/style.css

```diff
@@ -6,8 +6,11 @@
     text-align: center;
     background-color: #1a1414;
 }
+.text-block{
+    font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
```

Comment:
2 or three fonts should suffice.

Font should either be the found one or a fallback one, that's why Arial and sans serif are the last two ones.
```
font-family: 'Trebuchet MS', Arial, sans-serif;
```